ros_cross_compile/platform.py

```diff
@@ -61,7 +61,13 @@ class Platform:
     """
 
     def __init__(
-        self, arch: str, os_name: str, ros_distro: str, override_base_image: Optional[str] = None,
+        self,
+        arch: str,
+        os_name: str,
+        ros_distro: str,
+        *,
+        override_base_image: Optional[str] = None,
+        testing_version: bool = False,
```

Comment:
testing_version is not really clear. It seems we're enabling testing of the version of... something?

`use_shadow_testing_repository` for instance?